content/lua/concepts/conditionals/conditionals.md

```diff
@@ -0,0 +1,102 @@
+---
+Title: 'Conditionals' # Required; the file name should be the same as the title, but lowercase, with dashes instead of spaces, and all punctuation removed
+Description: 'Conditionals take an expression and evaluate it to `true` or `false`. Code blocks run depending on the outcome of the expression.' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

Comment:
```suggestion
Description: 'Conditionals are used to evaluate statements as true or false.  # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```

Reply:
there is no punctuation permitted within the quotes of metadata items

Reply:
```suggestion
Description: 'Conditionals are used to evaluate statements as true or false ' # Required; ideally under 150 characters and starts with a noun (used in search engine results and content previews)
```